src/applications/vaos/utils/address.js

```diff
@@ -52,14 +52,23 @@ export function calculateBoundingBox(lat, long, radius) {
   ];
 }
 
-export function getPreciseLocation() {
+/**
+ * Returns the current position of the device.
+ *
+ * @param enableHighAccuracy Indicates the application would like to receive the best possible results.
+ * @param maximumAge Indicates that the application is willing to accept a cached position whose age is no greater than the specified time in milliseconds.
+ * @returns {Object}
+ */
+export function getPreciseLocation(enableHighAccuracy = true, maximumAge = 0) {
```

Comment:
Based on your testing, I don't see a clear benefit to enabling high accuracy, and it's likely to increase the time it takes to find the location. I also don't think super high precision is all that necessary for searching for CC providers, since most will be miles away from you.

So I think I'd say we default `enableHighAccuracy` to false for now. What do you think @narin?

I still think it could be useful to warn people if we get a location back that is really low accuracy, but that'd be a separate set of work.

Reply:
I agree with Jeff.  The results of the accuracy testing were pretty surprising to me and there doesn't seem to be much benefit at the moment. Thanks for taking the time to test these out.